membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -86,7 +86,7 @@ class AccountController(
     AuthAndBackendViaAuthLibAction(requiredScopes = List(readSelf, updateSelf)).async { implicit request =>
       val tp = request.touchpoint
       val cancelForm = Form { single("reason" -> nonEmptyText) }
-      val maybeUserId = request.user.map(_.identityId)
+      val maybeUserId = request.user.toOption.map(_.identityId)
```

Comment:
also confused why we can discard the error message here

Reply:
At this point, authorisation has already been given so the `user` instance is just being used for the purposes of the endpoint.  If it wants to throw an error later it can.